pkg/controller/storagecluster/initialization_reconciler.go

```diff
@@ -297,6 +298,77 @@ func (r *ReconcileStorageCluster) newStorageClasses(initData *ocsv1.StorageClust
 	return ret, nil
 }
 
+// newRBDSnapshotClass returns a new RBD backed SnapshotClass
+func (r *ReconcileStorageCluster) newRBDSnapshotClass(initData *ocsv1.StorageCluster) *snapapi.VolumeSnapshotClass {
+	retSC := &snapapi.VolumeSnapshotClass{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: generateNameForRBDSnapshotClass(initData),
+		},
+		Driver: generateNameForRBDSnapshotClassDriver(initData),
+		Parameters: map[string]string{
+			"clusterID": initData.Namespace,
+			"csi.storage.k8s.io/snapshotter-secret-name":      "rook-csi-rbd-provisioner",
```

Comment:
`csi.storage.k8s.io/snapshotter-secret-name` can be a constant or var. thus secret namespace below too.

Reply:
Changed to constants...